variables.tf

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "image" {
+  description = "GCE image on which to base the Bastion. This image is supported by Shielded VM"
+  default = "gce-uefi-images/centos-7"
+}
+
+variable "instances" {
```

Comment:
Is this used anywhere in `main.tf`?

Reply:
Good catch, I started going down the path of intance groups and then rolled back